### ISSUE TITLE
  We downloaded the data set and model parameters in your steps, but get  the value of loss is Nan at run time.

### ISSUE THREAD

**Issue by aoxipo** [twke18/Adaptive_Affinity_Fields#13]
We downloaded the data set and model parameters in your steps, but get  the value of loss is Nan at run time. here is print ------   loss = nan, lr = 0.001000:   0%|          | 1/200 [00:49<2:44:03, 49.46s/it]，
 I would appreciate it if you could reply to me.

**Comment by fishfish2018** [comment]
I have the same problem. if you can share me your detail step to run model i will appreciate for you

**Comment by zhengtianyu1996** [comment]
Hi, have you solved this problem? I apply the affinity loss function to Deeplab, tensorflow=1.7.0, python=3.6, system=Windows. I met the same problem, the loss value is NaN. If you solved this problem, can you give me some advice?

**Comment by fishfish2018** [comment]
Sorry, I did not solve this problem.

**Comment by xychenunc** [comment]
Have you solved the problem and how? Same problem here

Thanks
